open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -38,14 +38,6 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
  * Developers can specify the environment variable(s) in a .env file, by copying [.env_sample](nodejs/.env_sample) to a file
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
-
-## FOR NEW DEVELOPMENT (as of 8/2/2020)
```

Comment:
May be an idea to use ISO format dates YYYY-MM-DD as we have a global audience and this is US date format

Reply:
changed date format